src/Cryptol/REPL/Command.hs

```diff
@@ -2004,15 +2004,13 @@ moduleInfoCmd isFile name
 --  */
 -- @
 extractCodeBlocks :: T.Text -> Either String [[T.Text]]
-extractCodeBlocks raw = go [] cleanLines
+extractCodeBlocks raw = go [] (T.lines raw)
   where
-    cleanLines = T.dropWhile isSpace <$> T.lines raw
-
     go finished [] = Right (reverse finished)
     go finished (x:xs)
       | (ticks, lang) <- T.span ('`' ==) x
       , 3 <= T.length ticks
-      = if lang == "repl"
+      = if lang `elem` ["", "repl"]
```

Comment:
It might be worth leaving a comment here explaining what the meaning of the empty string `""` is in this context.